### DISCUSSION TITLE
Function to archive subevents

### DISCUSSION THREAD

**Discussion by rp3r** [pretix/pretix#2118] [Ideas]
Hello

We have been using pretix successfully for many swimming pools since last year.
However, our customers are now increasingly asking for some kind of archive function for past subevents.
Is there anything planned in this direction? The problem is that the subevents from 2020 influence the statistics.

This would probably be relevant to all seasonal events.

**Comment by raphaelm** [comment]
I fully understand the need, we see similar issues with customers. A good solution would be to just do one event series per year, but I know that's not helpful in hindsight. 

The main problem with an archival function for both events and subevents is that we'd need to decide that that actually means, i.e. from which statistics or lists or views it will be hidden and from which statistics it won't, and we'll not only need to decide this once, but basically for every single view in the system. So the effort would be quite high, and the user interface confusion probably as well.

What statistics specifically are your customers struggling with?

**Reply by rp3r** [reply]
Insbesondere the dashboard and its Attendees (orderes + paid) and the total revenue.

Many of our customers used this page last year to get some quick info about their event.
